pkg/authorization/apis/authorization/conversion.go

```diff
@@ -241,3 +243,42 @@ func Convert_rbac_PolicyRules_To_authorization_PolicyRules(in []rbac.PolicyRule)
 	}
 	return rules
 }
+
+func copyMapExcept(in map[string]string, except string) map[string]string {
+	out := map[string]string{}
+	for k, v := range in {
+		if k != except {
+			out[k] = v
+		}
+	}
+	return out
+}
+
+func convert_Origin_Annotations_To_rbac_Annotations(in map[string]string) map[string]string {
+	// can't use const in pkg/oc/admin/policy because of import cycle
+	value, ok := in["openshift.io/reconcile-protect"]
+	if ok {
+		out := copyMapExcept(in, "openshift.io/reconcile-protect")
+		if value == "true" {
+			out[rbac.AutoUpdateAnnotationKey] = "false"
+		} else {
```

Comment:
So this makes a bit more sense to me:

```go
func convert_authorization_Annotations_To_rbac_Annotations(in map[string]string) map[string]string {
	// can't use const in pkg/oc/admin/policy because of import cycle
	if value, ok := in["openshift.io/reconcile-protect"]; ok {
		out := copyMapExcept(in, "openshift.io/reconcile-protect")
		if value == "true" {
			out[rbac.AutoUpdateAnnotationKey] = "false"
		}
		return out
	}
	return in
}

func convert_rbac_Annotations_To_authorization_Annotations(in map[string]string) map[string]string {
	if value, ok := in[rbac.AutoUpdateAnnotationKey]; ok {
		out := copyMapExcept(in, rbac.AutoUpdateAnnotationKey)
		if value == "false" {
			out["openshift.io/reconcile-protect"] = "true"
		}
		return out
	}
	return in
}
```

Because we only honor the annotation for a specific value.  Any other value is ignored (so its "true" or "...any other value..." for origin to RBAC, not "true" or "false").  It also makes it so that we do not add the (meaningless) `openshift.io/reconcile-protect: "false"` annotation to our bootstrap policy.

All that said, I will defer to @liggitt and @deads2k because I do not feel strongly enough either way.